ATB/DataAccess/Dtos/BookingDto.cs

```diff
@@ -0,0 +1,14 @@
+namespace DataAccess.Dtos;
```

Comment:
DataAccess.Dtos are not used, right ? I yhink you can delete them
 as this project is implemented using Layered Architecture.
 so business layer is using DataAccess.Models directly